### DISCUSSION TITLE
Send message not working: content-script <---> dom (Manifest v3)

### DISCUSSION THREAD

**Discussion by mdedirudianto** [quasarframework/quasar#12947] [CLI - BEX mode] [answered]
I tried new version of Quasar CLI BEX using manifest v3. It looks like that connection between content-script and dom has broken. Working fine with older version of Quasar webpack and BEX manifest v2. Please find my codes below to test

**dom.js:**
```js
console.log('dom file')

export default function (bridge) {
  console.log('attach dom bridge')

  bridge.on('test', d => {
    console.log('test from content', d.data)
  })

  console.log('send dom ready')
  bridge.send('dom-ready', { dom: true })
}
```

**my-content-script.js**
```js
console.log('content file')

export default function (bridge) {
  console.log('attach content')

  setInterval(async () => {
    console.log('send test to dom');
    await bridge.send('test', { test: true });
  }, 5000);

  bridge.on('dom-ready', (event) => {
    console.log('content dom-ready', event.data);
  })
}
```

No message received on both side. Is there anything I missed?

**Comment by mdedirudianto** [comment]
Updates:

Working with older version quasar/webpack:
```
 Dev mode.......... bex
 Pkg quasar........ v2.6.1
 Pkg @quasar/app... v3.3.3
 Pkg webpack....... v5
 Debugging......... enabled
```

With newest version, doesn't work:
```
 » Dev mode.................. bex
 » Pkg quasar................ v2.6.1
 » Pkg @quasar/app-webpack... v3.4.5
 » Transpiled JS..... yes (Babel)
```

**Reply by rstoenescu** [reply]
Can confirm the issue. Created a ticket: https://github.com/quasarframework/quasar/issues/12974
Please follow the resolution there.

**Comment by rstoenescu** [comment, marked as answer]
Can confirm the issue. Created a ticket: https://github.com/quasarframework/quasar/issues/12974
Please follow the resolution there.
